packages/core/src/transport/httpRequest.spec.ts

```diff
@@ -187,7 +187,7 @@ describe('httpRequest', () => {
       const onResponseSpy = jasmine.createSpy('xhrOnResponse')
 
       interceptor.withMockXhr((xhr) => {
-        const syntheticEvent = new Event('loadend')
+        const syntheticEvent = createNewEvent('loadend', {}, false)
```

Comment:
**💬 suggestion:** maybe leveraging the `properties` argument like ‏`createNewEvent('loadend', { __ddIsTrusted: false })` could be simpler/more explicit...